google/resource_cloudfunctions_function.go

```diff
@@ -180,6 +188,13 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 				ValidateFunc: validation.StringInSlice(allowedIngressSettings, true),
 			},
 
+			"vpc_connector_egress_settings": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
```suggestion
				Optional:     true,
				Computed:     true,
```

This field is conditionally set to `PRIVATE_RANGES_ONLY`, which Terraform can't express well. Marking as `Computed` or "ignore if unset" is most appropriate.